clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -194,4 +194,18 @@ public ApiVersionsResults apiVersions(Collection<Node> nodes) {
      * @return                  The ApiVersionsResults.
      */
     public abstract ApiVersionsResults apiVersions(Collection<Node> nodes, ApiVersionsOptions options);
+
+    public DescribeConfigsResults describeConfigs(Collection<ConfigResource> resources) {
```

Comment:
Sure, I did it for the AdminClient methods. I intend to do a Javadoc pass over all AdminClient classes, filed KAFKA-5274 to track that.